### PR TITLE
plotting method for GID15 dataset

### PR DESCRIPTION
**Pull request by nilsleh**
Added plotting method for GID15 dataset. 
Screenshot from 2021-12-16 21-33-24

### CODE REVIEW

File: torchgeo/datasets/gid15.py [hunk 1]
```diff
@@ -239,3 +240,38 @@ def _download(self) -> None:
...
+        self, sample: Dict[str, Tensor], suptitle: Optional[str] = None
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample return by :meth:`__getitem__`
+            suptitle: optional suptitle to use for figure
+
+        Returns;
+            a matplotlib Figure with the rendered sample
```
Comment (adamjstewart) [inline torchgeo/datasets/gid15.py:254]: Needs:
```rst
.. versionadded:: 0.2
```

**Comment by nilsleh** [comment]
> Needs to add support for predicted masks.

Sorry, I am not sure I understand. I thought the plot method is supposed to just show a sample from the dataset, and not any predictions?

**Comment by isaaccorley** [comment]
> Sorry, I am not sure I understand. I thought the plot method is supposed to just show a sample from the dataset, and not any predictions?

We are in the process of updating all plotting methods for datasets as well as adding the ability to subplot a predicted target if it's in the sample dict (e.g. under the key "predictions"). See https://github.com/microsoft/torchgeo/pull/264

**Comment by isaaccorley** [comment]
This looks good. Can you add some example plots to the PR description as a sanity check?

**Review by adamjstewart — APPROVED** [review]

**Comment by calebrob6** [comment]
Overriding the broken mypy check

**Comment by adamjstewart** [comment]
I think the key should be "prediction", not "predictions". None of our other keys (image, mask, label) are plural.

**Pull request merged**